$MirOS: ports/devel/automake/1.9/patches/patch-lib_am_ltlib_am,v 1.1.7.1 2005/03/18 15:44:28 tg Exp $
--- lib/am/ltlib.am.orig	Sat May 14 20:20:44 2005
+++ lib/am/ltlib.am	Sun Dec  6 15:15:49 2009
@@ -28,10 +28,10 @@ endif %?INSTALL%
 if %?INSTALL%
 am__installdirs += "$(DESTDIR)$(%NDIR%dir)"
 ?BASE?%DIR%LTLIBRARIES_INSTALL = $(INSTALL)
-?!BASE?%DIR%LTLIBRARIES_INSTALL = $(install_sh) -c
+?!BASE?%DIR%LTLIBRARIES_INSTALL = $(SHELL) $(install_sh) -c
 ?EXEC?.PHONY install-exec-am: install-%DIR%LTLIBRARIES
 ?!EXEC?.PHONY install-data-am: install-%DIR%LTLIBRARIES
-install-%DIR%LTLIBRARIES: $(%DIR%_LTLIBRARIES)
+install-%DIR%LTLIBRARIES:
 	@$(NORMAL_INSTALL)
 	test -z "$(%NDIR%dir)" || $(mkdir_p) "$(DESTDIR)$(%NDIR%dir)"
 ## Funny invocation because Makefile variable can be empty, leading to
